### ISSUE TITLE
unpack/repack of vbmeta.img doesn't work

### ISSUE THREAD

**Issue by Tkkg1994** [cfig/Android_boot_image_editor#24]
Hey :)

I've successfully unpacked my vbmeta.img and also repacked it. But one thing doesn't work:

As soon as you set algorithm_type = 0 you'll get this issue:

```
> Task :packClear FAILED
Exception in thread "main" java.io.IOException: Is a directory
        at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
        at sun.nio.ch.FileDispatcherImpl.read(FileDispatcherImpl.java:46)
        at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
        at sun.nio.ch.IOUtil.read(IOUtil.java:197)
        at sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:159)
        at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:65)
        at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:109)
        at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:103)
        at java.nio.file.Files.read(Files.java:3105)
        at java.nio.file.Files.readAllBytes(Files.java:3158)
        at cfig.Avb.packVbMeta(Avb.kt:324)
        at cfig.Avb.packVbMetaWithPadding(Avb.kt:374)
        at cfig.Avb.packVbMetaWithPadding$default(Avb.kt:373)
        at cfig.RKt.main(R.kt:19)

* What went wrong:
Execution failed for task ':packClear'.
> Process 'command '/usr/lib/jvm/java-8-openjdk-amd64/bin/java'' finished with non-zero exit value 1

```
Everything else works perfectly fine

**Comment by cfig** [comment]
oh yes, i reproduced this problem

**Comment by cfig** [comment]
Hi Tkkg1994,
I submitted a fix for this:

fix issue #3: packing error when "algorithm_type = 0"

Please have a try with latest 'master' branch.
